proposals/dao/tip_121c_pt2.ts

```diff
@@ -0,0 +1,112 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  PcvStats,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+
+/*
+
+Tribe Redemption 
+
+*/
+
+// Circulating amount of TRIBE, which is redeemable for underlying PCV assets
+// TODO: Update with final circulating TRIBE figure
+const REDEEM_BASE = ethers.constants.WeiPerEther.mul(458_964_340);
+
+// Lido deposit balance, being withdrawn and sent to Tribe Redeemer
+const STETH_DEPOSIT_BALANCE = '50296523674661485703301';
+
+const DAO_TIMELOCK_FOX_BALANCE = '15316691965631380244403204';
+const DAO_TIMELOCK_LQTY_BALANCE = '1101298805118942906652299';
+
+// Minimum DAI_HOLDING_DEPOSIT_BALANCEamount of DAI redeemable for Tribe (total DAI held - amount on FeiPSM)
+// TODO: Update with final numbers
+const DAI_HOLDING_DEPOSIT_BALANCE = ethers.constants.WeiPerEther.mul(30_600_000);
```

Comment:
TO CHECK : what's the final amount of DAI left in the protocol, that can be sent to the TribeRedeemer ?